mkt/prices/models.py

```diff
@@ -347,7 +347,13 @@ def create_addon_purchase(sender, instance, **kw):
                      c=instance))
         return
 
-    if instance.type == mkt.CONTRIB_PURCHASE:
+    print
+    print dir(instance.price_tier)
+    print instance.price_tier.price
+    print instance.price_tier.name
+
+    if instance.type == mkt.CONTRIB_PURCHASE \
+            and not instance.price_tier.name == '0':
```

Comment:
Hmm, I wonder if we should not save the AddonPurchase record for all in-app purchases.

I'm struggling to find out where we actually store the data that a user purchased an in-app product. We need to store that, especially for free products so they can be restored.